components/life-cycle-extension/src/main/java/com/wso2telco/dep/lifecycleextension/util/WorkflowProperties.java

```diff
@@ -0,0 +1,66 @@
+package com.wso2telco.dep.lifecycleextension.util;
+
+import com.wso2telco.core.dbutils.fileutils.PropertyFileReader;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class WorkflowProperties {
+
+    private static HashMap<String, String> propertiesMap = null;
+    private static final Log log = LogFactory.getLog(WorkflowProperties.class);
+
+    public static Properties loadWorkflowProperties(){
```

Comment:
what is the purpose of this method?

Reply:
now, this method was unuse I will resolve this .thank you